src/components/Causes/CausesDropInPaymentWrapper.vue

```diff
@@ -0,0 +1,175 @@
+<template>
```

Comment:
The plan is to reuse this wrapper for both Update and Register -- similar to how the monthly good drop in wrapper works